### ISSUE TITLE
Proposal: Sign new_balance_root and new_nullifier_root

### ISSUE THREAD

**Issue by jesusholoo** [agglayer/agglayer#241]
Super related to this issue https://github.com/agglayer/agglayer/issues/233

In order to have a "easier", more complete and compact signature i propose to sign:

Instead of:
https://github.com/agglayer/agglayer/blob/92159de06d6ea82c7cf7beeb13f53bb2a401a0ff/crates/pessimistic-proof/src/local_state.rs#L72-L75

```rust
        let combined_hash: [u8; 32] = keccak256_combine([
            multi_batch_header.new_local_exit_root.as_slice(),
            multi_batch_header.new_balance_root.as_slice(),
            multi_batch_header.new_nullifier_root.as_slice()
        ]);
```

**Comment by hadjiszs** [comment]
Yeah this was one of the option and we didn't go with this option because it implied having the execution on the balance tree and the nullifier tree on the CDK side, so that the new roots can be signed by the trusted sequencer of the CDK, but the CDK as of today maintains only the local exit tree but not the two other trees.

When these two new trees will be maintained on the CDK side (maybe with vnext?), then the payload that will be signed could be the 3 roots that you mentioned, and also the CDK will have everything to compute the PP by itself

**Comment by krlosMata** [comment]
Current approach aligns better if the PP submitter is a rollup/validium, where the `newLER` and the `commitImportedBirdgeExits` should be the output of the `stateTransistion` proof and the input of the `ppProof`
